add-chain/testdata/superchain/configs/sepolia/expected.yaml

```diff
@@ -0,0 +1,19 @@
+name: awesomechain
+chain_id: 42069
+public_rpc: http://awe.some.rpc
+sequencer_rpc: http://awe.some.seq.rpc
+explorer: https://awesomescan.org
+superchain_level: 2
+batch_inbox_addr: "0xff00000000000000000000000000000000042069"
+genesis:
+  l1:
+    hash: "0x1fc104fe244042459314bc1d51a230320c64a3d47cd06b7998aa3f57b9b0fc79"
+    number: 5753349
+  l2:
+    hash: "0x0ceefff2113580e4d665c324661e4982a824025bf97285cf09a52715b6f89083"
+    number: 0
+  l2_time: 1713792864
+canyon_time: 0
+delta_time: null
+ecotone_time: null
```

Comment:
This is fine for now, but will change when we tackle https://github.com/ethereum-optimism/superchain-registry/issues/166